packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -222,6 +222,12 @@ class BuildInfo {
   /// so the uncapitalized flavor name is used to compute the output file name
   String? get uncapitalizedFlavor => _uncapitalize(flavor);
 
+  /// The module system DDC is targeting, or null if not using DDC.
+  DdcModuleFormat? get ddcModuleFormat => _ddcModuleFormatFromFrontEndArgs(extraFrontEndOptions);
+
+  /// 'true' if this build uses the DDC module system.
+  bool get usesDdcModules => ddcModuleFormat == DdcModuleFormat.ddc;
```

Comment:
ubernit, but is there an actual readability gain in this getter over just inlining the comparison?

Reply:
Nope - beyond being slightly less verbose. I can just use direct comparisons if that's preferable; I'm not opinionated at all here.

Reply:
i'd slightly prefer inline comparisons--when reading this code, i would jump to the definition anyway to see what the logic was

Reply:
Sounds good - updated